specs/core/0_beacon-chain.md

```diff
@@ -1249,7 +1261,7 @@ def get_initial_beacon_state(initial_validator_deposits: List[Deposit],
         batched_block_roots=[],
 
         # Deposit root
-        latest_deposit_root=latest_deposit_root,
+        latest_pow_chain_data=latest_pow_chain_data,
         deposit_root_votes=[],
```

Comment:
```suggestion
        pow_chain_data_votes=[],
```

Reply:
I suggest changing `pow_chain_` to `eth1_` because the Ethereum 1.0 chain may not remain a PoW chain in the long term. Also shorter :)

Reply:
yeah, `eth1_` is more descriptive and succinct imo

Reply:
Done!

Reply:
Done!